src/wui/wui_vars.h

```diff
@@ -7,7 +7,7 @@
 #define Y_AXIS_POS 1
 #define Z_AXIS_POS 2
 typedef struct {
-    char gcode_name[GCODE_NAME_MAX_LEN + 1];
+    char *gcode_name;
```

Comment:
@joshymjose @XPila 
in case this var was previously used only as a temporary buffer to store the filename being printed, sholdn't it now be a const char * to prevent accidental write into it?

Reply:
this will break the code. original code keeps copy of the file change in WUI thread.

Reply:
at the line 83 in wui_api.c the web_vars.gcode_name used as char string. now it is pointer to char string. hopefully it does not break?

Reply:
it is pointer and by default is set to null - in this case matlin api will not handle notifications with filename. Clients that need receive filename notifications must allocate this pointer. This is done for wui in this pulrequest (struct web_client_t, member media_file_name).